src/constants/memory/wram.asm

```diff
@@ -3303,8 +3303,9 @@ wBoomerangTradedItem::
 wKidSaveHintIndex::
   ds 1 ; DB7E
 
-; Unlabeled
-wDB7F::
+; Set to $01 or $02 if you handed over your trade sequence item to someone and have not gotten the next one yet.
+; wTradeSequenceItem will still indicate where you are in the sequence, and wDoesNotHaveTradeSequenceItem will hide the item from your inventory menu.
+wDoesNotHaveTradeSequenceItem::
```

Comment:
To avoid the negative formulation, how about

```suggestion
wExchangingTradeSequenceItem::
```

?

(Feel free to discard this, of course)

Reply:
Ah, yes, that is better. I had some throuble thinking of a name for this